packages/sitecore-jss-proxy/.eslintrc

```diff
@@ -0,0 +1,7 @@
+{
+  "extends": "../../.eslintrc",
+  "rules": {
+    // TODO: around 190 errors
```

Comment:
Same here - I wouldn't consider this a TODO.

Reply:
Removed